docs/getting-started/setup/configuration.md

```diff
@@ -1,14 +1,9 @@
 ---
-# Scope:
-# * Introduction to setting configurations.
-# * Introduction to the top and must-know configurations.
-# * Point where to find the list of configuration options.
-
 category: setup
-menu-title: Configuration
+menu-title: Configuring features
 meta-title: Configuration | CKEditor 5 documentation
 meta-description: Learn how to configure CKEditor 5. 
-order: 10
+order: 20
 modified_at: 2024-05-06
 ---
 
```

Comment:
H1 needs to change.

Reply:
I wouldn't move people to framework in the paragraph of adding features. TBH, I would remove all links to framework, and remove the word build. Add a link to the features section where you could find all the features.

> Note that some features may require more than one plugin to run. You will find the plugin-per-feature listing in the {@link framework/architecture/plugins#plugins-and-html-output Plugins and HTML output} guide.

This also needs rewording and not linking to HTML output.

Reply:
In commit.